e2e/protractor.gh-firefox.conf.js

```diff
@@ -0,0 +1,19 @@
+// Protractor configuration file, see link for more information
+// https://github.com/angular/protractor/blob/master/lib/config.ts
+
+protractorBaseConfig = require('./protractor.base.conf');
+const config = protractorBaseConfig.config;
+
+config.multiCapabilities = [
+  {
+    browserName: 'firefox',
+    firefoxOptions: {
+      args: ['--headless']
+    },
+    'moz:firefoxOptions': {
```

Comment:
Clarification: do we need to specify the `--headless` option twice?

Reply:
Using just `firefoxOptions` does not work. Solutions on stack overflow suggested the second `moz:firefoxOptions` to be added in conjunction with the first one to ensure workability across all platforms.

Reply:
Ok. Can you add a comment recording this, with a link to the most relevant stackoverflow thread, directly above `firefoxOptions` ?

This is just to help future developers.

Reply:
Actually I just re-tested this and tried removing the first `firefoxOptions` this time and it seems to be able to run headless locally and in GH Actions? Shall I just leave it as that? Then there won't be a duplicate `--headless`? I can leave the documentation related to using `moz:firefoxOptions` as the default capability in the comments.

Reply:
Sure, let's remove the extra `--headless` option then.